x-pack/legacy/plugins/maps/public/actions/map_actions.d.ts

```diff
@@ -27,31 +31,20 @@ export function updateSourceProp(
   newLayerType?: LAYER_TYPE
 ): void;
 
-export interface MapCenter {
-  lat: number;
-  lon: number;
-  zoom: number;
-}
-
-export function setGotoWithCenter(config: MapCenter): AnyAction;
+export function setGotoWithCenter(config: MapCenterAndZoom): AnyAction;
 
 export function replaceLayerList(layerList: unknown[]): AnyAction;
 
 export interface QueryGroup {
```

Comment:
I'd make these a `type` to be consistent with how other these of data-objects are represented in Maps